troyleach/divegalaxsea-ui
# Added the home page picture gallery

### Changes that were made
- Added a picture gallery to the home page
- Added fake data for now because the API does not support images as of yet


### How to test drive PR:
- pull down the branch
- run `npm i` 
- spin up a server. should see 3 different pictures that change every 2 seconds
